QUEST.tsv

```diff
@@ -1184,22 +1184,22 @@ QUEST_20150317_001183	Having now lost the gold bullion given to him by the stran
+QUEST_20150317_001190	Modestas could see that the wagon was very durable, has been well cared for, and had a good appearance. However, he had no immediate need for a wagon, so he hesitated. {nl} In addition, Modestas had no money now; no matter how low the wagon owner went, he did not have the money to buy. {nl}
+QUEST_20150317_001191	The human heart sure can be an amusing thing. {nl} Modestas, deciding the wagon was worth buying but lacking the money to do so, decided he could use his impending victorious bet to win the wagon for free. {nl} So he proposed a bet with the wagon master, risking his own horse. {nl} Naturally, Modestas won. {nl}
+QUEST_20150317_001192	Modestas, having now just won a wagon and an injured horse, originally intended to keep just the wagon and his original horse. {nl} But seeing as the injured horse isn't as injured as he thought, he believed he had won big. {nl}
+QUEST_20150317_001193	In his mind, now that he's had obtained two horses and a wagon, he believed he no longer in a bad situation, deciding to this time be really careful not to lose big next. {nl} So, instead of following the soldiers, he decided to go to a place where there are less people. He reined his horse and travelled away from civilization. {nl} However, even in a remote setting, he was bound to meet some people. {nl}
+QUEST_20150317_001194	Rare as it would be, Modestas, while moving about in a people-less mountain, encountered an elderly Alchemist. {nl} The old Alchemist sat on a chair on the right side of the road, a silver nugget in hand. {nl} As Modestas passed by the Alchemist for a closer look, the nugget turned out to not be of silver, but of lead. {nl}
+QUEST_20150317_001195	"Could this Alchemist be one who could turn lead into gold?" thought Modestas, but later assuming he was just exaggerating. {nl} But the Alchemist, glad to see the first person pass by in a long time, called out to Modestas, and suddenly proposed a bet with him. Somehow, today the whole world wants to make bets with Modestas, and he was getting tired of it. {nl} Modestas liked gambling, but to find opponents this easily was becoming a joke to him. {nl}
+QUEST_20150317_001196	Modestas was about to do something unlike his being and reject the bet, but the idea that he could wager low moved his mind. {nl} After all, the old Alchemist was wagering a lead nugget. {nl}
+QUEST_20150317_001197	Because this time he was certain to lose, Modestas believed it would be better now to wager and lose against a lead nugget than to later run into people who would wager and lose a large amount of money. Modestas first made sure that he could accept the bet now despite having no money at hand and could bring some later, and accepted the bet. No words need to be said about the result. {nl}
```

Comment:
Should have been more clear, I think it's better to put the now here:
"Because this time he was certain to lose, Modestas believed it would be better to wager and lose against a lead nugget **now** than to later run into people who would wager and lose a large amount of money."